frontend/src/components/DashboardPage.js

```diff
@@ -142,8 +142,9 @@ const DashboardPage = ({
               </Col>
               <Col {...STATS_COL_PROPS}>
                 {((librariesSummary.library_type_counts && Object.keys(librariesSummary.library_type_counts)) || []).map((library_type) =>
-                  <Statistic title={libraryTypesByID[library_type]?.name} value={librariesSummary.library_type_counts[library_type] || "—"} />
+                  <Statistic title={libraryTypesByID[library_type]?.name || "-"} value={librariesSummary.library_type_counts[library_type] || "—"} />
```

Comment:
Why the additional dashes ? If you have libraries you should have library_types ... this is meant while it is loading ?

Reply:
Yes